broker-core/src/test/java/io/zeebe/broker/incident/ExpressionIncidentTest.java

```diff
@@ -351,4 +351,38 @@ public void shouldResolveIncidentForExclusiveGatewayWithoutMatchingCondition() {
     testClient.receiveFirstIncidentEvent(IncidentIntent.RESOLVED);
     testClient.receiveElementInState("workflow", ELEMENT_COMPLETED);
   }
+
+  @Test
+  public void shouldResolveIncidentIfInstanceCanceled() {
+    // given
+    testClient.deploy(
+        Bpmn.createExecutableProcess("workflow")
```

Comment:
This workflow is used multiple times in the test class. We should write it only once :)